README.md

```diff
@@ -23,6 +23,15 @@ The documentation is generated from the lagom main project as unstyled HTML snip
 
 ## Developing the website
 
+Due to bintray being shutdown, some of the dependencies of this build are not available for download from anywhere. The easiest way to address this is build them locally:
+
+```
+git clone https://github.com/lightbend/lightbend-markdown.git
+cd lightbend-markdown
+git checkout v1.7.0
+sbt publishLocal
+```
```

Comment:
@octonato I noticed you've built/deployed the docs yourself, is this what you did to get this dependency?

Reply:
I didn't build the docs, but only updated the release page. 

For the record, I did so because Vegemite is gone and I had to publish from my machine.